src/components/SlateEditor/plugins/embed/RelatedArticleBox.jsx

```diff
@@ -0,0 +1,214 @@
+/**
```

Comment:
Er det mulig å splitte opp denne komponenten i litt mindre deler?

Reply:
Tror det kan være fordelaktig å dele opp i en editering og en visning komponent. 

Synes også denne er litt uheldig og gjør koden tyngre å lese:

```js
 const relatedArticles = items.map(
+      (item, i) =>
+        !item.id ? (
+          'Invalid article'
+        ) : (
+          <div key={item.id} {...classes('article')}>
+            <RelatedArticle
+              {...mapping(resourceType(item).id)}
+              title={get(item, 'title.title')}
+              introduction={get(item, 'metaDescription.metaDescription')}
+              to={`/learning-resource/${item.id}/edit/${locale}`}
+            />
+            {editMode && (
+              <Button
+                stripped
+                onClick={e => this.removeArticle(i, e)}
+                {...classes('delete-button')}>
+                <Cross />
+              </Button>
+            )}
+          </div>
+        ),
+    );
```
Bedre å duplisere litt kode for kun gjøre editMode sjekken et sted.